### ISSUE TITLE
Payment Methods Names

### ISSUE THREAD

**Issue by goncalocruz77** [KingIsulgard/Magento-Noble-Extended-Order-Grid#4]
Hi, I have a problem with the form, but I have a question about how the names of the payment methods appear, the names appear as the tags of the developers. change the field from Shipping Method to Shipping Description the names began to appear as they are in the checkout.But with the payment methods do not have the Payment Description column, then the names appear as in the image:

![FoxSea](https://user-images.githubusercontent.com/14839168/57804813-efb17c80-7731-11e9-955b-e78a5550537b.JPG)

How can I process these fields through the module? It would be to choose the names of the forms of payment. Choose the names that would appear instead of the names that are appearing.

It would be interesting if it appeared in the place of these Tags the formats that we use (Credit Card, Boleto Bancaria, Two Credit Cards, Debit Card, Bank Transfer ...).

Thank you for your help.

**Comment by KingIsulgard** [comment]
Yes the payment description wasn't available directly in the order grid.
You can add it if you add a little lookup script that matches the tag with
the correct description.

I didn't add that because it would make the order grid slower for a not so
important reason.

The code of the module is freely available on github so feel free to tweak
it.

Regards,
Gilles

Op wo 15 mei 2019 om 21:55 schreef goncalocruz77 <notifications@github.com>:
